server/src/routes/trpc.ts

```diff
@@ -0,0 +1,57 @@
+import { initTRPC } from "@trpc/server";
+import { createExpressMiddleware } from "@trpc/server/adapters/express";
+import { z } from "zod";
+import { getDb } from "~/db/lib/get-db";
+
+const createContext = () => {
+  return {
+    ...getDb(process.env.DATABASE_URL!),
+  };
+};
+
+const t = initTRPC.context<typeof createContext>().create();
+
+const router = t.router;
+
+const trpcRouter = router({
+  getStudentById: t.procedure
+    .input(z.object({ student_id: z.string().uuid() }))
+    .query(async (req) => {
+      const { student_id } = req.input;
+
+      const result = await req.ctx.db
+        .selectFrom("student")
+        .where("student_id", "=", student_id)
+        .selectAll()
+        .executeTakeFirst();
+
+      return result;
```

Comment:
It would be good to be explicit about the core types in our system like `student`. We can define interfaces for those (either explicitly or via protobufs ;) and import those on the client.

Adding a TODO for this is ok for now.